ui/src/components/VoxPrompts.js

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {LitElement, html} from 'lit-element';
+
+import * as promptApi from '../utils/PromptApiService';
+
+export class VoxPrompts extends LitElement {
+    static get properties() {
+        return {
+            prompt: {type: String},
+            state: {type: String},
+        };
+    }
+
+    constructor() {
+        super();
+        this.state = 'NOT_ASKED';
```

Comment:
Make this state an "enum"

https://www.sohamkamani.com/blog/2017/08/21/enums-in-javascript/